hoot-core-test/src/test/cpp/hoot/core/algorithms/MaximalSublineTest.cpp

```diff
@@ -248,8 +251,8 @@ class MaximalSublineTest : public CppUnit::TestFixture
       false, Status::Unknown1);
     MapProjector::projectToPlanar(map);
 
-    WayPtr w52 = dynamic_pointer_cast<Way>(TestUtils::getElementWithNote(map, "-52"));
-    WayPtr w812 = dynamic_pointer_cast<Way>(TestUtils::getElementWithNote(map, "-812"));
+    WayPtr w52 = boost::dynamic_pointer_cast<Way>(TestUtils::getElementWithNote(map, "-52"));
+    WayPtr w812 = boost::dynamic_pointer_cast<Way>(TestUtils::getElementWithNote(map, "-812"));
```

Comment:
Instead of `using namespace boost` I found it more "readable" with adding `boost::` before the pointer cast.